src/controllers/authorsController.js

```diff
@@ -0,0 +1,14 @@
+const { createItem, getAllItems, getItemByPk, updateItem, deleteItem } = require('../controllers/helpers');
```

Comment:
When destructuring a lot of values, its much more readable to import them with one per line like so:
```suggestion
const {
  createItem,
  getAllItems,
  getItemByPk,
  updateItem,
  deleteItem,
} = require("../controllers/helpers");
```

The same approach would be good to use for the module.exports at the end of the file